include/MFAnalog.h

```diff
@@ -28,7 +30,8 @@ class MFAnalog
 {
 public:
     MFAnalog(uint8_t pin = 1, analogEvent callback = NULL, const char * name = "Analog Input", uint8_t sensitivity = 2);
-    void update();    
+    void update();
+    void tick(void);   
```

Comment:
To keep consistent with the rest of the code this can omit `void` in the function parameters.

Reply:
Yes, changed it.